src/app/app-routing.module.ts

```diff
@@ -1,10 +1,22 @@
 import { NgModule } from '@angular/core';
-import { Routes, RouterModule } from '@angular/router';
+import { RouterModule, Routes } from '@angular/router';
 
-const routes: Routes = [];
+const routes: Routes = [
+  {
+    path: '',
+    redirectTo: '/users-search',
```

Comment:
what the idea then here to use lazy loading ?

Reply:
For consistency. I guess each of three feature modules (Users, ReposList, RepoInfo) should be lazy-loaded.

Reply:
got you. just to be aware, that, if I remember right, that lazy loaded modules have their own DI, so if we rely on services used across, we have to take into account this. Sorry, haven't had time to look more precisely, but look good, great job 👍